src/App.css

```diff
@@ -4,59 +4,46 @@
   box-sizing: border-box;
 }
 
+html {
+  font-size: 62.5%;
```

Comment:
why is the reason you added this? what are you trying to accomplish with this line of CSS?